### DISCUSSION TITLE
Hide cards without dependencies?

### DISCUSSION THREAD

**Discussion by Mashinji** [solidify/scale#25] [Q&A]
Hi @Scale,

I have created like 7 teams, and the dependencies works great - even created a new WorkItem called "dependency" and shows up fine with dependencies across the board.

However, when using the "Hide cards without dependencies" everything disappears except one Item that doesn't - this Item doesn't show any dependencies though.

So my question is, what is triggers the "hide cards without dependencies" functionality?

I have this enabled:
![image](https://github.com/solidify/scale/assets/22957310/fbf0c366-c54e-4c63-b2a7-74a720e70bdc)

I find it very great to only show the items with dependencies, but annoying when everything disappears :)

Hope someone can help me out... :)

**Comment by daniel-ohrn** [comment]
Hey @Mashinji!

Thanks for reaching out. 

The logic for "Hide cards without dependencies" is as follows: 

Cards that have a predecessor or successor relation to another card that is on the board stays - the cards that doesn't are hidden. 

Could you provide us with a screen shot of the item that is showing and one of an item that should be showing but ins't (when "Hide cards without dependencies" is turned on).
